push_data_to_ccos/get_community_team_data.py

```diff
@@ -94,6 +95,33 @@ def generate_databag():
     return databag
 
 
+def sort_databag(databag):
+    """     
+    This function orderes the member according to their roles
+    """
+    project_priority = {"Project Maintainer": 1, "Project Core Committer": 2,
+                        "Project Collaborator": 3, "Project Contributor": 4, }
+    community_builders_priority = {
+        "Community Collaborator": 1, "Community Contributor": 2}
+
+    for first_order_key in databag:
+        if first_order_key == 'projects':
+            for i in range(len(databag['projects'])):
```

Comment:
Using the for-in loop would be more Pythonic as
```suggestion
            for project in databag['projects']:
```

Reply:
thanks, done as you said.